src/config/config.go

```diff
@@ -31,13 +31,17 @@ type Config struct {
 
 	// Path of BTC addresses JSON file
 	BtcAddresses string `mapstructure:"btc_addresses"`
+	// Path of ETH addresses JSON file
+	EthAddresses string `mapstructure:"eth_addresses"`
```

Comment:
Validate this value in `Config.Validate()`

Reply:
done